### PR TITLE
new gene list for push to production

### LINKED ISSUE #30: push to production
need to push to production to correspond with new data release

### PR DESCRIPTION
**Pull request by raynamharris**
i created a new gene list from production and used it to crate a new list of gene pages to annotate. 

saved as `data/inputs/PRODUCTION__available_genes__2022-07-18.txt`

then i removed a handful of genes that had warnings such as `WARNING: missing markdown for identifier ENSG00000133136`

updated the export command to `DERIVA_SERVERNAME=app.nih-cfde.org` 

all worked with well the make commands but I'm getting this error 

```
line 46, in get_archive_headers_map
    raise ValueError('Could not obtain tokens for host "%s"' % host)
ValueError: Could not obtain tokens for host "app.nih-cfde.org"
```

### CODE REVIEW

**Comment by ctb** [comment]
I think you'll need to do the login step - `python3 -m cfde_deriva.cfde_login login app.nih-cfde.org`

**Comment by raynamharris** [comment]
one error fixed. getting a new error 

```
raise exception.ReleaseUnknown('Release "%s" not found in registry.' % (id,))
cfde_deriva.exception.ReleaseUnknown: Release "c8a13a7a-37d6-4fc4-b3fc-10e694ff24bb" not found in registry.
```

**Comment by ctb** [comment]
should be able to find the latest release number here,

https://app-dev.nih-cfde.org/chaise/recordset/#registry/CFDE:release@sort(RCT::desc::,RID)

which is taking a long time for me to load, actually

**Comment by raynamharris** [comment]
batch submission working thanks to https://github.com/nih-cfde/cfde-deriva/pull/370 

ready for approval and merge :)

File: Snakefile [hunk 1]
```diff
@@ -18,7 +18,7 @@ rule upload:
         "upload_json/gene.json",
         "upload_json/anatomy.json",
     shell: """
-        export DERIVA_SERVERNAME=app-staging.nih-cfde.org
+        export DERIVA_SERVERNAME=app.nih-cfde.org
```
Comment (ctb) [inline Snakefile]: this should not be hard-coded in since it will lead to heartbreak and misery...

**Comment by ctb** [comment]
I think we need more configuration options here, so we can configure for staging or production dynamically. But I worry a lot that we're going to mistakenly update production. Give me a bit of time to think of a good solution pls :)

**Comment by raynamharris** [comment]
I commented out the export command and added the other two options (also commented out). it would be cool if you could say something like "make update dev" or make update production" and the the correct export command would be uses to specify the correct server

**Review by jessicalumian — APPROVED** [review]

**Comment by raynamharris** [comment]
will close without merging. note, might need to recheck gene lists when pushing to production

**Pull request closed without merging**